src/h2load.cc

```diff
@@ -669,6 +781,31 @@ void Client::on_header(int32_t stream_id, const uint8_t *name, size_t namelen,
       }
     }
 
+    if (warmup == 2) {
```

Comment:
Yeah, also thought of this but again might bump into same problem I stated earlier. In some functions they do some _essential_ stuff which we might not want to touch. But yes, we can selectively implement this option. I'll do that.